build/bazelbuild.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
How about we move this file into `build/bazelbuilder` and add it in the builder image directly? Then it's more clear from the location of the file what it is used for.

Reply:
Done.